sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -4760,4 +4760,36 @@ ALTER TABLE campaignformdata_history ADD COLUMN archived boolean;
 
 INSERT INTO schema_version (version_number, comment) VALUES (224, 'Add archived column to campaign form data #2268');
 
--- *** Insert new sql commands BEFORE this line ***
\ No newline at end of file
+-- 2020-07-07 Adds visit to cases
+
+ALTER TABLE cases ADD COLUMN followupstatus varchar(255);
+ALTER TABLE cases ADD COLUMN followupcomment varchar(4096);
+ALTER TABLE cases ADD COLUMN followupuntil timestamp;
+ALTER TABLE cases ADD COLUMN overwritefollowupuntil boolean;
+
+UPDATE cases SET overwritefollowupuntil = false;
+
+ALTER TABLE cases_history ADD COLUMN followupstatus varchar(255);
+ALTER TABLE cases_history ADD COLUMN followupcomment varchar(4096);
+ALTER TABLE cases_history ADD COLUMN followupuntil timestamp;
+ALTER TABLE cases_history ADD COLUMN overwritefollowupuntil boolean;
+
+CREATE TABLE cases_visits(
```

Comment:
This table is not necessary since we don't need a many-to-many relation between cases and visits.

Reply:
OK